driver/cmd/ibm-spectrum-scale-csi/main.go

```diff
@@ -71,3 +75,11 @@ func createPersistentStorage(persistentStoragePath string) error {
 	}
 	return nil
 }
+
+func deleteStalePluginDir(stalePluginPath string) error {
+        if err := os.RemoveAll(stalePluginPath); err != nil {
```

Comment:
As a style point, can't we just return err and not have a branching conditional for the return?